apis/discoveryengine/v1alpha1/engine_types.go

```diff
@@ -66,10 +66,6 @@ type DiscoveryEngineEngineSpec struct {
 	// solution_type is SOLUTION_TYPE_SEARCH.
 	SearchEngineConfig *Engine_SearchEngineConfig `json:"searchEngineConfig,omitempty"`
 
-	// Optional. Whether to disable analytics for searches performed on this
-	//  engine.
-	DisableAnalytics *bool `json:"disableAnalytics,omitempty"`
```

Comment:
This looks unrelated.  Should this be in this PR?

Reply:
Yes, because we change the proto from v1 to v1alpha.